pkg/workloads/containerd/watcher.go

```diff
@@ -94,6 +94,7 @@ func listenForDockerEvents(ws *watcherState, reader io.ReadCloser) {
 		var e dTypesEvents.Message
 		if err := json.Unmarshal(scanner.Bytes(), &e); err != nil {
 			log.WithError(err).Error("Error while unmarshalling event")
+			continue
```

Comment:
Let's add a time.Sleep of 100ms just to prevent the cpu to reach 100% if we always read bad input.